kvbc/include/merkle_tree_key_manipulator.h

```diff
@@ -53,10 +54,16 @@ class DBKeyManipulator {
   static sparse_merkle::Hash extractHashFromLeafKey(const Key &key);
 
   // Extract the stale since version from a stale node index key.
-  static sparse_merkle::Version extractVersionFromStaleKey(const Key &key);
+  static sparse_merkle::Version extractVersionFromProvableStaleKey(const Key &key);
+
+  // Extract the stale since block id from a stale node index key.
+  static BlockId extractBlockIdFromNonProvableStaleKey(const Key &key);
 
   // Extract the actual key from the stale node index key.
-  static Key extractKeyFromStaleKey(const Key &key);
+  static Key extractKeyFromProvableStaleKey(const Key &key);
```

Comment:
Nit: As above, can be called `extractKeyFromTreeStaleKey`. Just a suggestion, please only change if you think it is better.